logging/nxscope/sensors/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# ##############################################################################
+# apps/logging/nxscope/sensors/CMakeLists.txt
```

Comment:
@raiden00pl do you think that apps/testing or apps/system is better to host sensor scope than apps/logging/nxscope/sensors? Since apps/logging/nxscope/sensors is a pure library before this patch.

Reply:
I agree, moved to apps/system